store/mongo/datastore_mongo.go

```diff
@@ -276,26 +279,52 @@ func (db *DataStoreMongo) DeleteTokenByDevId(ctx context.Context, devId string)
 }
 
 func (db *DataStoreMongo) Migrate(ctx context.Context, version string) error {
-	m := migrate.SimpleMigrator{
-		Session: db.session,
-		Db:      ctxstore.DbFromContext(ctx, DbName),
-	}
+	l := log.FromContext(ctx)
 
-	ver, err := migrate.NewVersion(version)
-	if err != nil {
-		return errors.Wrap(err, "failed to parse service version")
+	dbs := []string{DbName}
+
+	if db.multitenant {
+		l.Infof("running migrations in multitenant mode")
+
+		tdbs, err := migrate.GetTenantDbs(db.session, DbName)
```

Comment:
So, with comments from https://github.com/mendersoftware/go-lib-micro/pull/28#pullrequestreview-53131833 this will become:
```
tdbs, err := migrate.GetTenantDbs(db.session, store.IsTenantDb(DbName))
...
```